### DISCUSSION TITLE
Is there a way to make chips in TextEdit?

### DISCUSSION THREAD

**Discussion by vauradkar** [emilk/egui#5448] [Q&A] [answered]
Hi,

I’m new to egui and really appreciate the amazing work being done. In Material UI, you can group a set of words in a TextBox into visually distinct 'chips.' Is something similar possible in egui? If so, could you please guide me on how to implement it?

Thanks,
Vikram

**Comment by lucasmerlin** [comment, marked as answer]
There is no build in widget for this, but you should be able to do it in a horizontal_wrapped, each chip could be a label with a frame and the last item is the text edit

**Reply by vauradkar** [reply]
Sounds good. Thank you so much for the responding.
